src/tests/unit/tests/background/stores/tab-store.test.ts

```diff
@@ -107,27 +121,33 @@ describe('TabStoreTest', () => {
     });
 
     test.each`
-        payloadUrl                               | initialIsOriginChanged | expectedIsOriginChanged
-        ${'https://original-host/original-path'} | ${false}               | ${false}
-        ${'https://original-host/new-path'}      | ${false}               | ${false}
```

Comment:
Are we removing the test cases for `https://original-host/new-path` here because those are being tested in `url-parser.test.ts` now?

Reply:
Yes - `https://original-host/original-path` and `https://original-host/new-path` are both the same origin as `https://original-host/original-path`, so those two cases collapse into one now that the tab-store logic is reduced to checking "are the two URLs the same origin or not?"